core/src/test/scala/cell/base.scala

```diff
@@ -461,39 +692,47 @@ class BaseSuite extends FunSuite {
     pool.shutdown()
   }
 
-  test("whenNextComplete: dependency 1") {
+  test("whenNextSequential: complete dependent cell") {
```

Comment:
Why did you modify an existing test? Wouldn't it be sufficient to just add new tests?